stories/MemoryRouter.js

```diff
@@ -0,0 +1,20 @@
+export class MemoryRouter {
+  constructor(initialState = {}) {
+    this._memoryState = initialState;
+  }
+  write(routeState) {
+    this._memoryState = routeState;
+  }
+  read() {
+    return this._memoryState;
+  }
+  createURL() {
+    return '';
+  }
+  onUpdate() {
+    return {};
+  }
+  dispose() {
+    this._memoryState = {};
```

Comment:
We still have an issue with the `MemoryRouter`. 

The behavior is different when we land directly on the story or if we navigate through it.

Reply:
Fixed it by creating one MemoryRouter instance per story, it should do the trick.